auth/auth_test.go

```diff
@@ -125,3 +134,62 @@ func TestSignatureTxMiddlewareMultipleTxSameBlock(t *testing.T) {
 	NonceTxPostNonceMiddleware(state, nonceTxBytes, loomchain.TxHandlerResult{}, nil)
```

Comment:
Take another look at this test it seems `NonceTxPostNonceMiddleware` is always getting `nonceTxBytes`, not what was passed into `NonceTxHandler.Nonce`/`NonceTxMiddleware`

Reply:
Fixed